### ISSUE TITLE
Suggestion: Decimal places always needed

### ISSUE THREAD

**Issue by mzvarik** [vuejs-tips/v-money#25]
I want 2 decimal places.

But when I type "1", it says "0.01"

So  to input "1" I have to type 3 keys!

This is really annoying...

Except that, this is the best component ever.

**Comment by neves** [comment]
That's true, but I couldn't find a solution for it.
Do you have any idea?

**Comment by mzvarik** [comment]
I am new to Vue, I am pretty much lost... I am now using the Vue component here, that works, but it just adds decimals:
https://jsfiddle.net/chrisvfritz/1oqjojjx/?utm_source=website&utm_medium=embed&utm_campaign=1oqjojjx

Also this one is nice:
http://www.decorplanit.com/plugin/

**Comment by MoacirSchmidt** [comment]
Guy, this is the best masked input for vue I have ever seen! Unfortunately, "Decimal places always needed" suggestion is a "must have" enhancement....I have no idea how to implement these vue components, but do you have access to low level javscript routines to implement that?

**Comment by AndreasHerss** [comment]
I Agree, this is the only thing that is keeping me from using this package :(

**Comment by roncemer** [comment]
Yes, there should be an option to lose the implied decimal point.  Just let the user enter text free-form, then when the input loses focus, convert it to a number, round it to the correct scale, then put the rounded and formatted number back into the input and update the model's value accordingly.

**Comment by ghost** [comment]
I have the same issue
